### PR TITLE
fix(react): remove typo around `"" 2`

### PR DESCRIPTION
**Pull request by vvo**
linked to #213

**Summary**

**Result**

### CODE REVIEW

File: packages/react-instantsearch-theme-algolia/mixins/_searchbox.shipow.scss [hunk 1]
```diff
@@ -123,7 +123,7 @@
       margin-right: -4px;
       height: 100%;
       vertical-align: middle;
-      content: ''2
+      content: ''
```
Comment (Haroenv) [inline packages/react-instantsearch-theme-algolia/mixins/_searchbox.shipow.scss]: I think it was supposed to be a semicolon

**Comment by mthuret** [comment]
@oliviertassinari, how snapshots made by argos are updated? Will closing this PR enough?

**Comment by oliviertassinari** [comment]
@mthuret You can always accept the diffs in order to geta green PR but that won't impact future PRs. We consider the screenshots accepted once merged. Also, Argos-CI tries to find the screenshots corresponding to the fork commit between the base branch and the compare one. For instance.

**Comment by oliviertassinari** [comment]
When he can't find such fork commit, he fallback to the latest taken screenshots of the compare branch.

**Comment by oliviertassinari** [comment]
The idea is to get a behavior as close as possible as if the screenshots where stored in git.

**Pull request merged**